pages/overview/what-is-jsonschema.md

```diff
@@ -6,36 +6,287 @@ title: What is JSON Schema?
+
+
+### JSON Schema to the rescue
+
+JSON Schema provides a solution to this problem. It is a specification language for JSON that allows you to describe the structure, content, and semantics of a JSON instance. With JSON Schema, you can define metadata about an object's properties, specify whether fields are optional or required, and define expected data formats.
+
+By using JSON Schema, people can better understand the structure and constraints of the JSON data they are using. It enables applications to validate data, ensuring it meets the defined criteria. With JSON Schema, you can make your JSON more readable, enforce data validation, and improve interoperability across different programming languages.
+
+Using the same example, we can validate the data by making use of the [jsonschema](https://github.com/python-jsonschema/jsonschema) Python library:
+```python
+from jsonschema import validate
+
+# Using JSON Schema
+data = {
+    "product": {
+        "name": "Widget",
+        "price": 10.99,
+        "quantity": 5
+    }
+}
+
+schema = {
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "title": "Product",
+    "type": "object",
+    "properties": {
+        "product": {
+            "type": "object",
+            "properties": {
+                "name": {
+                    "type": "string"
+                    },
+                "price": {
+                    "type": "number", 
+                    "minimum": 0
+                    },
+                "quantity": {
+                    "type": "integer", 
+                    "minimum": 1
+                    }
+            },
+            "required": ["name", "price", "quantity"]
+        }
+    },
+    "required": ["product"]
+}
+
+try:
+    validate(data, schema)
+    print("Valid JSON object.")
+except Exception as e:
+    print("Invalid JSON object:", e)
+
+```
+
+In the above code snippet, we defined a JSON Schema that describes the expected structure and constraints of the JSON data. We use the `jsonschema` library to validate the `data` against the `schema`. If the data doesn't conform to the schema, an exception is raised, providing detailed information about the failure.
+
+By using JSON Schema, we can easily define and enforce constraints, making the validation process more robust and manageable. It improves the readability of the code and reduces the chances of data-related issues.
+
+
+### Steps to Validate JSON Data Using JSON Schema
+
+1. **Create the schema**
+
+    You begin by defining the rules in a JSON Schema document using [keywords](../learn/glossary.md/#keyword).
```

Comment:
You can't link to just the markdown file like that 😅.
For links, either the full URL, or preferably a relative URL from the root `/learn/glossary#keyword`.

```suggestion
    You begin by defining the rules in a JSON Schema document using [keywords](/learn/glossary/#keyword).
```

Reply:
I think I made this mistake in one other place as well, I'll just fix it 😅